insonmnia/hub/config.go

```diff
@@ -67,6 +69,10 @@ type Config struct {
 	MetricsListenAddr string             `yaml:"metrics_listen_addr" default:"127.0.0.1:14000"`
 }
 
+func (c *Config) LogLevel() (zapcore.Level, error) {
+	return logging.ParseLogLevel(c.Logging.Level)
```

Comment:
Can't we parse this in `NewConfig` instead to produce errors in construction time instead of runtime? In some private field, which won't affect deserialization.
It saves some lines of code.